src/main/java/com/faforever/api/config/IgnoreOctetStreamToObjectHttpMessageConverter.java

```diff
@@ -0,0 +1,40 @@
+package com.faforever.api.config;
+
+import com.faforever.api.map.MapUploadMetadata;
+import org.springframework.http.HttpInputMessage;
+import org.springframework.http.HttpOutputMessage;
+import org.springframework.http.MediaType;
+import org.springframework.http.converter.AbstractHttpMessageConverter;
+import org.springframework.lang.Nullable;
+import org.springframework.util.StreamUtils;
+
+import java.io.IOException;
+
+public class IgnoreOctetStreamToObjectHttpMessageConverter extends AbstractHttpMessageConverter<byte[]> {
```

Comment:
Please some javadoc what this does and why it's needed :)

Reply:
done